symbiosis/resource_cluster.go

```diff
@@ -62,6 +56,32 @@ func ResourceCluster() *schema.Resource {
 					},
 				},
 			},
+			"endpoint": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Cluster API server endpoint",
+			},
+			"identity": {
```

Comment:
As discussed on GH we can probably remove this layer and put the cert and key on ther too level